ethminer/MinerAux.h

```diff
@@ -309,6 +309,22 @@ class MinerCLI
 				}
 			}
 		}
+                else if (arg == "--cuda-parallel-hash")
+                {
+                        try {
+                                m_parallelHash = stol(argv[++i]);
+                                if(m_parallelHash == 0 || m_parallelHash>8)
```

Comment:
No. The maximum value is 8, and is limited by the current implementation. The detailed information can be found at libethash-cuda/dagger_shuffled.cuh